tso_session/intro_TSO_data_pipeline.ipynb

```diff
@@ -136,11 +136,74 @@
     "First the imports.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import astropy.io.ascii as ascii
</pre><p>Please move this import to the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>done</p>